packages/dashboard/src/app/routes/_authenticated/_orders/components/order-table-totals.tsx

```
@@ -58,7 +61,19 @@ export function OrderTableTotals({ order, columnCount }: Readonly<OrderTableTota
             </TableRow>
             <TableRow>
                 <TableCell colSpan={columnCount - 1} className="h-12">
-                    <Trans>Shipping</Trans>
+                    <div className="flex flex-wrap gap-1">
+                        <div>
+                            <Trans>Shipping</Trans>
+                        </div>
+                        {order.shippingLines.map(sl => (
+                            <Badge variant="outline">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)